packages/nx-plugin/src/executors/cli/executor.int.test.ts

```diff
@@ -38,11 +42,17 @@ describe('runAutorunExecutor', () => {
         projectConfig: expect.objectContaining({ name: 'utils' }),
       }),
     );
-    // eslint-disable-next-line n/no-sync
-    expect(execSync).toHaveBeenCalledTimes(1);
-    // eslint-disable-next-line n/no-sync
-    expect(execSync).toHaveBeenCalledWith(expect.stringContaining('utils'), {
-      cwd: process.cwd(),
-    });
+    expect(executeProcessSpy).toHaveBeenCalledTimes(1);
+    expect(executeProcessSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        command: 'npx',
+        args: expect.arrayContaining(['@code-pushup/cli']),
+        cwd: process.cwd(),
+        observer: expect.objectContaining({
+          onError: expect.any(Function),
+          onStdout: expect.any(Function),
+        }),
+      }),
+    );
```

Comment:
The `expect.objectContaining` wrappers aren't needed because there are no other properties.

```suggestion
    expect(executeProcessSpy).toHaveBeenCalledWith({
      command: 'npx',
      args: expect.arrayContaining(['@code-pushup/cli']),
      cwd: process.cwd(),
      observer: {
        onError: expect.any(Function),
        onStdout: expect.any(Function),
      },
    });
```

There are 2 such occurrences in `executor.unit.test.ts` also, see subsequent comments.